conans/client/generators/b2.py

```diff
@@ -223,25 +222,25 @@ def b2_variation(self):
             }.get(self.conanfile.settings.get_safe('compiler'))+'-'+self.b2_toolset_version
             self._b2_variation['architecture'] = {
                 'x86': 'x86', 'x86_64': 'x86',
-                'ppc64le': 'power', 'ppc64': 'power',
-                'armv6': 'arm', 'armv7': 'arm', 'armv7hf': 'arm', 'armv8': 'arm',
-                'armv7s': 'arm', 'armv7k': 'arm',
+                'ppc64le': 'power', 'ppc64': 'power', 'ppc32': 'power',
+                'armv6': 'arm', 'armv7': 'arm', 'armv7hf': 'arm', 'armv7s': 'arm', 'armv7k': 'arm',
+                'armv8': 'arm', 'armv8_32': 'arm', 'armv8.3': 'arm',
                 'sparc': 'sparc', 'sparcv9': 'sparc',
                 'mips': 'mips1', 'mips64': 'mips64',
             }.get(self.conanfile.settings.get_safe('arch'))
             self._b2_variation['instruction-set'] = {
                 'armv6': 'armv6', 'armv7': 'armv7', 'armv7hf': None, 'armv7k': None,
-                'armv7s': 'armv7s', 'armv8': None, 'avr': None,
+                'armv7s': 'armv7s', 'armv8': None, 'armv8_32': None, 'armv8.3': None, 'avr': None,
                 'mips': None, 'mips64': None,
-                'ppc64le': None, 'ppc64': 'powerpc64',
+                'ppc64le': None, 'ppc64': 'powerpc64', 'ppc32': None,
                 'sparc': None, 'sparcv9': 'v9',
                 'x86': None, 'x86_64': None,
             }.get(self.conanfile.settings.get_safe('arch'))
             self._b2_variation['address-model'] = {
```

Comment:
The address-model setting for ppc32 was left out.

Reply:
Where can I find this information? Would it be ``32``, ``powerpc32`` or just ``powerpc``? Thanks for the review

Reply:
I imagine it's just 32.

Reply:
I have included this in the new changes. Thanks a lot!